README.md

```diff
@@ -1,3 +1,27 @@
 [ ![Codeship Status for databraid-dashboard/slack-api](https://app.codeship.com/projects/623bb170-6021-0135-e845-2e6fa0f02ff3/status?branch=master)](https://app.codeship.com/projects/239348)
 
 # Databraid Slack API
+
```

Comment:
@kurtishouser I think we should have one newline here after the H1 instead of 3.